src/Data/List/Properties.agda

```diff
@@ -758,6 +758,10 @@ length-take zero    xs       = refl
 length-take (suc n) []       = refl
 length-take (suc n) (x ∷ xs) = cong suc (length-take n xs)
 
+take-one-more : {ℓ : Level} {X : Set ℓ} {m : ℕ} (x : Fin m) (f : Fin m → X) →
+   take (toℕ x) (tabulate f) ∷ʳ f x ≡ take (suc (toℕ x)) (tabulate f)
```

Comment:
I think this equality should the other way around. We usually have the function application on the left and the reduction behaviour on the right.

Reply:
The meta-heuristic being
```
complex => simple(r)
```